build.gradle

```diff
@@ -28,11 +28,30 @@ repositories {
     mavenCentral()
 }
 
+def agronaVersion = '1.4.0'
+def aeronVersion = '1.26.0'
+def jmhVersion = '1.23'
+def hdrHistogramVersion = '2.1.12'
+def disruptorVersion = '3.4.2'
+def junitVersion = '5.6.0'
+def mockitoVersion = '3.3.3'
+
 dependencies {
-    annotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess:1.23'
-    implementation 'org.openjdk.jmh:jmh-core:1.23'
-    implementation 'io.aeron:aeron-all:1.26.0'
-    implementation 'com.lmax:disruptor:3.4.2'
+    api "org.agrona:agrona:${agronaVersion}"
+    
+    annotationProcessor "org.openjdk.jmh:jmh-generator-annprocess:${jmhVersion}"
+    implementation "org.openjdk.jmh:jmh-core:${jmhVersion}"
+    implementation "io.aeron:aeron-archive:${aeronVersion}"
```

Comment:
Any reason not to stick with aeron-all?

Reply:
I have `HdrHistogram` and `Agrona` as the dependencies. `aeron-all` has `HdrHistogram` as relocated dependency (i.e. it under a different package name `io.aeron.shadow.org.HdrHistogram`). But `Agrona` is included as is. It just seemed the cleanest to pull normal Aeron jars and add all required dependencies explicit.

The options here are:
- Use `aeron-all` and rely on it to provide the dependencies we need, i.e. use `io.aeron.shadow.org.HdrHistogram` instead of `org.HdrHistogram`.
- Use `aeron-all` and all only `HdrHistogram` dependency relying on the `Agrona` classes being present in the Aeron jar.
- Use `aeron-archive` (or maybe `aeron-cluster`) as the dependency and add all other libs explicitly.